### PR TITLE
Make sure default variant is always used in CI end-2-end tests

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/488551667048375/1204340991927492/f

### Description
This PR makes sure the default variant is always used when running end to end tests on CI

### Steps to test this PR

- [x] Build the app normally and a variant should be allocated to your
- [x] Build the app using `-Pforce-default-variant`, for example `./gradlew installInternalDebug -Pforce-default-variant` and make sure you are always assigned the default variant.
- [x] All CI tests should also pass in this PR
- [x] All tests in https://github.com/duckduckgo/Android/actions/runs/4637681307 should pass and if they are not is not because the onboarding cannot be finished

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3045** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3045" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3045?utm_source=stack-comment).

File: app/src/main/java/com/duckduckgo/app/buildconfig/RealAppBuildConfig.kt [hunk 1]
```diff
@@ -52,6 +52,8 @@ class RealAppBuildConfig @Inject constructor() : AppBuildConfig {
         }
     }
     override val isPerformanceTest: Boolean = BuildConfig.IS_PERFORMANCE_TEST
+
+    override val forceDefaultVariant: Boolean = BuildConfig.FORCE_DEFAULT_VARIANT
```
Comment (aitorvs) [inline app/src/main/java/com/duckduckgo/app/buildconfig/RealAppBuildConfig.kt:56]: rename to `isDefaultVariantForced`?

**Review by aitorvs — APPROVED** [review]
Left one comment about naming of a val that checks boolean value.

Otherwise LGTM and works as expected

File: app/src/main/java/com/duckduckgo/app/buildconfig/RealAppBuildConfig.kt (hunk 1 above)
Comment (marcosholgado) [reply app/src/main/java/com/duckduckgo/app/buildconfig/RealAppBuildConfig.kt]: Done!

**Pull request merged**